vmc/resource_vmc_sddc.go

```diff
@@ -96,11 +398,22 @@ func resourceSddcCreate(d *schema.ResourceData, m interface{}) error {
 				}
 				return resource.RetryableError(fmt.Errorf("instance creation still in progress"))
 			}
+			// Resiliency in case of difficulties the VMC service may experience,
+			// during the significant SDDC provisioning time
+			if err.Error() == (errors.ServiceUnavailable{}.Error()) {
+				serviceUnavailableRetries++
+				if serviceUnavailableRetries <= maxServiceUnavailableRetries {
+					return resource.RetryableError(fmt.Errorf(
```

Comment:
Perhaps we should give it a few seconds between each retry to have a chance to recover?

Reply:
This code is executed inside a function provided to the RetryContext. That function is called every 10 second (the default).